### PR TITLE
graphql@14.0 peer dependencies, revert greenkeeper-lockfile 

### PR DESCRIPTION
**Pull request by acao**
Hopefully gets us closer to #244

## Greenkeeper
Should unblock #251 , which appeared to fail because `greenkeeper-lockfile@1` was not aware of the `-W` flag for root-level yarn workspaces changes

https://github.com/greenkeeperio/greenkeeper-lockfile#---greenkeeper-now-has-built-in-support-for-updating-lockfiles--

reverts previously external `greenkeeper-lockfile` project to use greenkeeper 3's native lockfile support:
https://blog.greenkeeper.io/announcing-native-lockfile-support-85381a37a0d0

because greenkeeper 3 also introduces lerna support (also yarn workspaces presumably, though they dont mention it the monorepo article?), this may cause some unexpected behavior in terms of existing/new PRs for this project once it's merged?

## Graphql 14.0 Peers
this also bumps the `graphql` peer dependencies across the projects. 

## Graphql Config
Bumped it to 2.2.1

## Yarn lockfile integrity hashes
Seems the last time the lockfile was built, there were no integrity sha checksums

### CODE REVIEW

**Comment by acao** [comment]
@AGS- wanna merge this?

File: packages/graphql-language-service/package.json [hunk 1]
```diff
@@ -29,11 +29,11 @@
     "prepublish": "node ../../resources/prepublish.js"
   },
   "peerDependencies": {
-    "graphql": "^0.6.0 || ^0.7.0 || ^0.8.0-b || ^0.9.0 || ^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0"
+    "graphql": "^0.6.0 || ^0.7.0 || ^0.8.0-b || ^0.9.0 || ^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^0.14.0"
```
Comment (benjie) [inline packages/graphql-language-service/package.json]: You mean `^14.0.0` here and elsewhere in this PR - GraphQL jumped from `0.13` to `14.0`.

Comment (acao) [reply packages/graphql-language-service/package.json]: thanks for that, just saw you review the same issue in a `graphiql` PR yesterday

**Comment by acao** [comment]
@benjie closing this in favor of a PR that only bumps peer dependencies

**Pull request closed without merging**